tests/test_version_leq.py

```diff
@@ -0,0 +1,81 @@
+# Copyright 2020 - 2021 MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import itertools
+import unittest
+
+from parameterized import parameterized
+
+from monai.utils import version_leq
+
+
+# from pkg_resources
+def _pairwise(iterable):
+    "s -> (s0,s1), (s1,s2), (s2, s3), ..."
+    a, b = itertools.tee(iterable)
+    next(b, None)
+    return zip(a, b)
+
+
+# from pkg_resources
+torture = """
+    0.80.1-3 0.80.1-2 0.80.1-1 0.79.9999+0.80.0pre4-1
+    0.79.9999+0.80.0pre2-3 0.79.9999+0.80.0pre2-2
+    0.77.2-1 0.77.1-1 0.77.0-1
+    """
+
+TEST_CASES = (
```

Comment:
Great test cases. I think it covers all the cases we have in MONAI so far, @ericspod what do you think?

Thanks.

Reply:
> 
> 
> Looks good to me.
> @ericspod could you please help double confirm it?
> 
> Thanks.

It's fine by me, except note that this removes the code for preventing the duplicate warnings.